src/Infrastructure/Symfony/Command/MELImportCommand.php

```diff
@@ -18,15 +21,20 @@
 class MELImportCommand extends Command
 {
     public function __construct(
+        private LoggerInterface $logger,
         private MELExecutor $executor,
+        private DateUtilsInterface $dateUtils,
     ) {
         parent::__construct();
     }
 
     public function execute(InputInterface $input, OutputInterface $output): int
     {
+        $reporter = new Reporter($this->logger);
+        $now = $this->dateUtils->getNow();
+
         try {
-            $report = $this->executor->execute(laterThan: new \DateTimeImmutable('now'));
+            $report = $this->executor->execute(laterThan: $now, reporter: $reporter);
```

Comment:
```suggestion
            $report = $this->executor->execute(laterThan: $now, $reporter);
```

Je pense que c'est aussi la config de ton IDE :smile:

Reply:
Là pour le coup c'est parce qu'on ne pouvait pas mettre d'argument sans keyword après un keyword argument (`laterThan: $now`)